.github/workflows/coverage.yml

```diff
@@ -51,6 +63,8 @@ jobs:
           go-version-file: "go.work"
       - uses: taiki-e/install-action@cargo-llvm-cov
       - uses: taiki-e/install-action@nextest
+      - name: Fetch proof params
+        run: cargo run --bin forest-tool --no-default-features -- fetch-params --keys
```

Comment:
Would it be faster to fetch them via docker? I don't think there's a need to compile it unless we're reusing the same compilation units in the codecov stage compilation.

Reply:
True, I can switch to docker